plugins/node/instrumentation-amqplib/src/amqplib.ts

```diff
@@ -22,31 +22,36 @@ import {
   SpanKind,
   SpanStatusCode,
   ROOT_CONTEXT,
-} from '@opentelemetry/api';
+} from "@opentelemetry/api";
```

Comment:
Please revert these unrelated formatting changes

Reply:
Done - I would advise y'all think about adopting something like prettier though - I notice the formatting is not consistent through out the codebase.

Reply:
fully agree here. In the past gts was used but removed because of compatibility issues with other tools. I think noone found the time to setup it again.